src/pybind/mgr/volumes/fs/fs_util.py

```diff
@@ -0,0 +1,102 @@
+import os
+import errno
+import logging
+
+import cephfs
+import orchestrator
+
+from .exception import VolumeException
+
+log = logging.getLogger(__name__)
+
+def create_pool(mgr, pool_name):
+    # create the given pool
+    command = {'prefix': 'osd pool create', 'pool': pool_name}
+    r, outb, outs = mgr.mon_command(command)
+    if r != 0:
```

Comment:
drop?

Reply:
sure -- this commit is a part of pr #31763 -- I'll fix it there